spec/model/user_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tresor', photo: 'hhtps/manzi.com', bio: 'this is manzi user', postsCounter: 2) }
```

Comment:
- [ ] Please ensure that any custom methods in the models have a corresponding unit test, So make sure you add a unit test for the `recent_posts` method in your user model :blush: